packages/w3/bin.js

```diff
@@ -1,41 +1,24 @@
 #!/usr/bin/env node
-import meow from 'meow'
-import w3 from './index.js'
 
-const cli = meow(
-  `Usage
-    --get bafybeidd2gyhagleh47qeg77xqndy2qy3yzn4vkxmk775bg2t5lpuy7pcu --output pics --api http://127.0.0.1:8787
-    --put /path/to/files
-  `,
-  {
-    importMeta: import.meta,
-    flags: {
-      api: {
-        type: 'string',
-        default: 'https://api.web3.storage'
-      },
-      token: {
-        type: 'string',
-      },
-      get: {
-        type: 'string',
-      },
-      put: {
-        type: 'string',
-      },
-      output: {
-        type: 'string',
-        alias: 'o',
-      }
-    },
-  }
-)
+import sade from 'sade'
+import { get, put } from './index.js'
 
-if (!cli.flags.get && !cli.flags.put) {
-  cli.showHelp()
-}
-if (cli.flags.get && cli.flags.put) {
-  cli.showHelp()
-}
+const cli = sade('w3')
 
-w3(cli)
+cli
+  .option('--api', 'URL for API to use')
+  .option('--token', 'API token to use')
```

Comment:
Not blocking, but we should have something like `w3 config set token <AUTH_TOKEN>` that writes to a `.w3rc` file in your home directory.

Reply:
Agree! The plan is to use [`conf`](https://github.com/sindresorhus/conf)  which recommends storing things in

>    macOS: ~/Library/Preferences/MyApp-nodejs
>    Windows: %APPDATA%\MyApp-nodejs\Config (for example, C:\Users\USERNAME\AppData\Roaming\MyApp-nodejs\Config)
>    Linux: ~/.config/MyApp-nodejs (or $XDG_CONFIG_HOME/MyApp-nodejs)

and the command to set it would be `w3 token <token>` for fewer words.

For CI world, we could have it check a .w3 file in the cwd too.